src/main/java/com/puppycrawl/tools/checkstyle/PackageObjectFactory.java

```diff
@@ -306,7 +305,7 @@ private Object createObjectFromFullModuleNames(String name, Set<String> fullModu
                         }));
         }
         catch (IOException ignore) {
-            returnValue = new HashMap<>();
+            returnValue = Collections.emptyMap();
```

Comment:
No mix for functional update and minor refactoring.
Please move to separate commit of the same PR.

Reply:
All changes to `PackageObjectFactory` were reverted.